docs/game/screens/影像店营业.md

```
@@ -79,9 +79,47 @@ source_image: screens/影像店营业/经营状况.webp
 - 流转:点 `录像带入口` 进 → app `check_recommended`(推荐上架 / 上架筛选)→ `click_filter` 点上架筛选 → `choose_theme` 选主题(`主题筛选` OCR + `difflib` 匹配 16 主题,找不到上拖 `drag_to -100`)→ `choose_onshelf` 上架(`上架`/`下架` 切换,点 2 次关筛选)→ 全部上架 → `返回`。
 
+### 4. 开始营业-确认态(source_image: screens/影像店营业/开始营业-确认.webp)
+
+- 出现:经营状况页(未营业 + 已选宣传员 + 已上架录像带)点「开始营业」→ 此确认弹窗(居中模态遮罩,背景变暗)。
+- 元素:「是否准备营业?」;「取消」(左,红 × 图标)/「确认」(右,绿 √ 图标)。
+- 匹配画面:`影像店营业`(is_precise=false)。
+- 匹配 area:
+
+  | area | 类型 | 文本 | 位置 |
+  |---|---|---|---|
+  | 开始营业-确认 | text | 确认 | 1032,600-1232,656 |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)